pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -2528,6 +2528,26 @@
         %rejected  ``noun+!>((~(has in rejected.cors-registry) u.origin))
       ==
     ::
+        [%scry @ @ ~]
+      =/  host  `i.t.tyl
+      =/  url  i.t.t.tyl
+      =/  server  (per-server-event [eny *duct now rof] server-state.ax)
+      =/  [=action suburl=@t]
+        (get-action-for-binding:server host url)
+      =/  invalid  (error-page 400 %.y url "invalid scry: {<action>} {<suburl>}")
```

Comment:
I don't think you want to actually produce an error page anywhere. Instead you want to produce `~` or `[~ ~]` for the failure cases ("unknown/invalid scry path" and "no result" respectively). Giving a proper http-event result would make the runtime serve that up, but if the scry can't handle it then we just want to fall through to the eventful handling, right?

Reply:
yes, all of this was just to make sure it was getting to the eyre vane. if you `!!` here it will take the normal flow.